rdrf/rdrf/db/contexts_api.py

```diff
@@ -123,3 +123,10 @@ def get_context(self, context_id, patient_model):
             return rdrf_context_model
         except RDRFContext.DoesNotExist:
             raise RDRFContextError("Context does not exist")
+
+    def get_previous_contexts(self, context_id, patient_model):
+        content_type = ContentType.objects.get_for_model(patient_model)
+        return RDRFContext.objects.filter(
+            pk__lt=context_id, registry=self.registry_model,
+            content_type=content_type, object_id=patient_model.pk,
+        ).order_by("-pk")
```

Comment:
Also, I think I would like it better if we did the ordering and `__lt` based on the `created_at` instead of the `pk`.

Reply:
Updated.